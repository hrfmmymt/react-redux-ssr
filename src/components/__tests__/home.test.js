// import React from 'react'
// import test from 'ava'
// import { shallow } from 'enzyme'
// import Home from '../home'

// test('ava test', () => {
//   const wrapper = shallow(<Home />)
//   wrapper.contains(<span>hoge</span>)
// })

import React from 'react'
import { shallow } from 'enzyme'
import Home from '../home'

test('jest test', () => {
  const wrapper = shallow(<Home />)
  // wrapper.contains(<span>hoge</span>)
  expect(wrapper).toMatchSnapshot()
})
