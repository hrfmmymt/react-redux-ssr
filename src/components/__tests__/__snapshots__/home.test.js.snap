// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest test 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Home />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          某のホームページ
        </h2>,
        <span>
          ★★★　最新情報　★★★
        </span>,
      ],
      "className": "home",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "某のホームページ",
        },
        "ref": null,
        "rendered": "某のホームページ",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "★★★　最新情報　★★★",
        },
        "ref": null,
        "rendered": "★★★　最新情報　★★★",
        "type": "span",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            某のホームページ
          </h2>,
          <span>
            ★★★　最新情報　★★★
          </span>,
        ],
        "className": "home",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "某のホームページ",
          },
          "ref": null,
          "rendered": "某のホームページ",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "★★★　最新情報　★★★",
          },
          "ref": null,
          "rendered": "★★★　最新情報　★★★",
          "type": "span",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
